packages/orchestrator-ui-components/src/types/subscription.ts

```diff
@@ -19,3 +19,7 @@ export type Subscription = {
     status: SubscriptionStatus;
     product: Pick<ProductDefinition, 'name' | 'tag' | 'productType'>;
 };
+export type SubscriptionListItem = {
```

Comment:
This type is already defined in "packages/orchestrator-ui-components/src/components/WFOSubscriptionsList/types.ts"

Reply:
And if I am correct this type is not in use anymore